web/tests/drafts.test.js

```diff
@@ -414,16 +414,25 @@ test("catch_buggy_draft_error", () => {
     const draft_model = drafts.draft_model;
 
     // An error is logged but the draft isn't fixed in this codepath.
-    blueslip.expect("error", "Cannot compare strings; at least one value is undefined");
-    drafts.rename_stream_recipient(
-        stream_B.stream_id,
-        "old_topic",
-        stream_A.stream_id,
-        "new_topic",
+    assert.throws(
+        () =>
+            drafts.rename_stream_recipient(
+                stream_B.stream_id,
+                "old_topic",
+                stream_A.stream_id,
+                "new_topic",
+            ),
+        {
+            name: "ZodError",
+        },
+    );
+
+    assert.throws(
```

Comment:
This now throws when getting invalid drafts, when it didn't before. Do we feel okay with that?